src/internal/vendor/d3-scale-third-party-licenses.txt

```diff
@@ -0,0 +1,220 @@
+Name: d3-array
```

Comment:
Is this file expected to be in git? It is generated by rollup, isn't it?

Reply:
Yes, it is generated by rollup. I added it to the repository where I added a snapshot test which fails incase of any changes in the licences content. This mechanism is intended to make us aware of any licence changes in the bundled 3rd party packages.

Reply:
Yes, I get why there is snapshot in git. But why `.txt` file too?

Reply:
That's because the licence plugin first attaches the banner followed by extracting the licence information of the bundled packages. Because of this order it would fail if the pre-generated licence file is not present. 

Assuming there are changes in the 3p licences happening during a build, the tests would fail because the generated file would contain the latest version. I'd love to have the licence extraction before the banner get's attached. I'm looking into the plugin to see why this is the case to understand better and consider about opening a PR for the licence plugin to get that solved.

Reply:
That's because the licence plugin first attaches the banner followed by extracting the licence information of the bundled packages. Because of this order it would fail if the pre-generated licence file is not present. 

Assuming there are changes in the 3p licences happening during a build, the tests would fail because the generated file would contain the latest version. I'd love to have the licence extraction before the banner get's attached. I'm looking into the plugin to see why this is the case to understand better and consider about opening a PR for the licence plugin to get that solved.